%YAML: 1.0

# 主要相机参数
#-----------------------------------------------
# flag_largeImg: 是否使用大图模式
# imgSizeL: 720p分辨率
# imgSizeS: 360p分辨率
# exposureTime: 曝光时间
#-----------------------------------------------
flag_largeImg: false
imgSizeL: 65537
imgSizeS: 131074
exposureTime: 2000